### PR TITLE
Code in the dark Editor

### PR DESCRIPTION
**Pull request by joaniffer**
Code in the dark editor/page completed:
image

Features
- success/failure messages
- not allowed to submit code if no changes made
- no code completion (auto complete ending tags, but that's it)

### CODE REVIEW

File: src/scenes/Editor/index.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+import 'ace-builds/src-noconflict/mode-html';
+import 'ace-builds/src-noconflict/theme-monokai';
+
+export default class Editor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const starterCode = `<html>
+  <head>
```
Comment (hackillinois-dev) [inline src/scenes/Editor/index.jsx:16]: Ahhhhh tab this over please

File: src/scenes/Editor/index.jsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      changeSinceSubmit,
+      successfulSubmit,
+      codeValue,
+    } = this.state;
+
+    const displayStatus = !successfulSubmit || !changeSinceSubmit;
+
+    return (
+      <div>
+        <h1>Code in the Dark!</h1>
```
Comment (hackillinois-dev) [inline src/scenes/Editor/index.jsx]: Delete the "!", put the Caterpillar logo somewhere on the page too!

File: src/scenes/Editor/index.jsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+        <div className="content">
```
Comment (hackillinois-dev) [inline src/scenes/Editor/index.jsx:103]: Give this some left margin, judging by the picture it's touching the edge of the screen

File: src/scenes/Editor/index.jsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            <h3>Rules:</h3>
+            <ol>
+              <li>Do not edit this code until we tell you to</li>
+              <li>Do not navigate away from this tab!</li>
+              <li>Do not preview your HTML code</li>
+            </ol>
+
+            <h3>Tips:</h3>
+            <ul>
+              <li>Our logo can be found at &quot;assets/logo.png&quot;</li>
```
Comment (hackillinois-dev) [inline src/scenes/Editor/index.jsx]: This might need to be updated with other assets... we will see

File: src/scenes/Editor/style.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  background: #272822;
+}
+
+h1, h3, ol, ul, p {
+  color: lightgray;
+  margin: 15px;
+}
+
+p {
+  margin: 0 0 0 15px;
```
Comment (hackillinois-dev) [inline src/scenes/Editor/style.scss]: margin-left: 15px;

**Review by hackillinois-dev — COMMENTED** [review]
Yesss looks very good!

File: src/scenes/Editor/index.jsx (hunk 1 above)
Comment (joaniffer) [reply src/scenes/Editor/index.jsx:16]: Isn't tabbed because the `` is literal and would reflect extra indenting in the editor

**Review by brianstrauch — APPROVED** [review]
💯

**Pull request merged**